Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -32,6 +32,7 @@ u32 position_matrix_index[3];
 typedef std::unordered_map<PortableVertexDeclaration, std::unique_ptr<NativeVertexFormat>> NativeVertexFormatMap;
 static NativeVertexFormatMap s_native_vertex_map;
 static NativeVertexFormat* s_current_vtx_fmt;
+u32 g_current_components;
```

Comment:
Maybe move the comment from https://github.com/dolphin-emu/dolphin/pull/3218/files#diff-5ff1a45ad75bac713802503f8b4ef4f5L118 to here?